project-chip/connectedhomeip
# Actions cluster test support

#### Problem
Support for testing actions clusters is missing.

#### Change overview
Support for testing action clusters was added to the bridge-app.

#### Testing
The updates to bridge-app were tested using chip-tool

The following command can be used to get the action cluster endpoint list
    chip-tool bridgedactions read endpoint-list 12344321 1

1. Start bridge app and wait for initialization to complete
2. Run chip-tool pairing
    chip-tool pairing ethernet 12344321 20202021 3840 localhost 5540
3. Run chip-tool to verify endpoint lists have 2 devices in "Room 1" and 2 devices in "Room 2"
4. Press r in the bridge-app to rename "Room 1" to "Room 1 renamed" (TC-ACT-2.2 step 2c)
6. press f in the bridge-app to move "Action Light 3" from "Room 2" to "Room 1" (TC-ACT-2.2 step 2f)
7. press i in the bridge-app to remove "Room 2" from exposed endpoint lists (TC-ACT-2.2 step 2i)
8. press l in the bridge-app to add "Zone 1" and include "Action Light 2" in the new zone (TC-ACT-2.2 step 2l)
